.coveragerc

```diff
@@ -3,6 +3,8 @@ branch = True
 omit = 
        *gen/jaeger*
        *gen/opencensus*
+       opencensus/__init__.py
```

Comment:
This is a workaround: the `__init__.py` file got replaced by another namespace package, which confused the coverage test.